src/Security/Authentication/test/OpenIdConnect/OpenIdConnectTests.cs

```diff
@@ -294,6 +295,50 @@ public async Task SignOut_WithMissingConfig_Throws()
             Assert.Equal("Cannot redirect to the end session endpoint, the configuration may be missing or invalid.", exception.Message);
         }
 
+        [Fact]
+        public async Task SignOut_WithMissingIssuer()
+        {
+            var settings = new TestSettings(o =>
+            {
+                o.SignInScheme = CookieAuthenticationDefaults.AuthenticationScheme;
+                o.Authority = TestServerBuilder.DefaultAuthority;
+                o.ClientId = "Test Id";
+            });
+            var server = settings.CreateTestServer(handler: async context =>
+            {
+                var claimsIdentity = new ClaimsIdentity("Cookies");
+                claimsIdentity.AddClaim(new Claim("iss", "test"));
+                await context.SignInAsync(new ClaimsPrincipal(claimsIdentity));
+            });
+
+            var signInTransaction = await server.SendAsync(DefaultHost);
+
+            var remoteSignOutTransaction = await server.SendAsync(DefaultHost + "/signout-oidc", signInTransaction.AuthenticationCookieValue);
+            Assert.Equal(HttpStatusCode.OK, remoteSignOutTransaction.Response.StatusCode);
```

Comment:
You can assert the absence of the Set-Cookie header that was skipped.
https://github.com/aspnet/AspNetCore/pull/6378/files#diff-77dbc837ce88da9bcf6a630ae8969190R164